src/specs/component.app.spec.ts

```diff
@@ -42,8 +40,11 @@ let debugElement: DebugElement
 let nativeElement: HTMLElement;
 
 describe('AppComponent', () => {
-  before(() => TestBed.configureTestingModule(testModuleConfig));
-  after(() => getTestBed().resetTestingModule());
+  beforeEach(() => TestBed.configureTestingModule(testModuleConfig));
```

Comment:
Do we really need to do this before each test? The config doesn't change, does it?

Reply:
not sure off the top of my head...

Reply:
seems to be needed in a before each.